### ISSUE TITLE
Clarify re-identification through dataset merger

### ISSUE THREAD

**Issue by darobin** [w3ctag/privacy-principles#98]
In #92, @mnot notes:

> Also:
>>    If a person could reasonably be identified or re-identified when data is combined with other data, then that makes that data personal data.
>
> is a bit meaningless; if I combine a phone book with a list of fruit at the local store, that makes the fruit personal data...

**Comment by jyasskin** [comment]
The text is still in https://w3ctag.github.io/privacy-principles/#acting-on-data, but there is other text on a similar topic in https://w3ctag.github.io/privacy-principles/#deidentified-data. Maybe we can just delete the bad text.

**Comment by darobin** [comment]
So, this was supposedly fixed in 5854be3b6b12a15c979d16af0089353db36a034b:

```
-If a [=person=] could reasonably be identified or re-identified when [=data=] is combined with other
-[=data=], then that makes that data [=personal data=].
+If a [=person=] could reasonably be identified or re-identified through the combination of [=data=] with other
+[=data=], then that [=data=] is [=personal data=].
```

The text that @mnot mentions is no longer in, it now says that it is only application when the identification happens _through_ the combination. I think we're fine?
